chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
-so that any material can be found when it is needed.
-Data structures need to be organized,
-so that the various steps in creating datasets
-can always be traced and revised without massive effort.
-The structure of version histories and backups need to be organized,
-so that different workstreams can exist simultaneously
-and experimental analyses can be tried without a complex workflow.
-Finally, the outputs need to be organized,
-so that it is clear what results go with what analyses,
-and that each individual output is a readable element in its own right.
-This chapter outlines how to stay organized
-so that you and the team can focus on getting the work right
-rather than trying to understand what you did in the past.
+Transforming raw data into a substantial contribution to scientific knowledge 
+requires a mix of subject expertise, programming skills, 
+and statistical and econometric knowledge. 
+The process of data analysis is, therefore, 
+a back-and-forth discussion between people 
+with differing experiences, perspectives, and research interests. 
+The research assistant usually ends up being the pivot of this discussion. 
+It is their job to translate the data received from the field into
+economically meaningful indicators and to analyze them 
+while making sure that code and outputs do not become tangled and lost over time.
+
+When it comes to code, though, analysis is the easy part, 
+\textit{as long as you have organized your data well}. 
+Of course, there is plenty of complexity behind it: 
+the econometrics, the theory of change, the measurement methods, and so much more.
+But none of those are the subject of this book. 
+Instead, this chapter will focus on how to organize your data work.
+Most of a Research Assistant's time is spent cleaning data and getting it into the right format. 
+When the practices recommended here are adopted,
+analyzing the data is as simple as using a command that is already implemented in a statistical software. 
+
+
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Organizing primary survey data}
+\section{Data management}
+The goal of data management is to organize the components of data work 
+so it can traced back and revised without massive effort.
+In our experience, there are four key elements to good data management: 
+folder structure, task breakdown, master scripts, and version control. 
+A good folder structure organizes files so that any material can be found when needed.
+It reflects a task breakdown into steps with well-defined inputs, tasks, and outputs.
+This breakdown is applied to code, data sets, and outputs.
+A master script connects folder structure and code.
+It is a one-file summary of your whole project.
+Finally, version histories and backups enable the team 
+to edit files without fear of losing information.
+Smart use of version control also allows you to track 
+how each edit affects other files in the project.
+
+% Task breakdown
```

Comment:
Follow the order in the intro paragraph: folder structure, task breakdown, master scripts, version control (so move this section to after folder structure, or change the intro paragraph)

Reply:
Related, labeled subsections would help organize content / guide the reader

Reply:
Done in 17174e8